MBBSEmu/Session/Rlogin/RloginSession.cs

```diff
@@ -121,9 +121,9 @@ protected override (byte[], int) ProcessIncomingClientData(byte[] clientData, in
             {
                 if (ProcessIncomingByte(clientData[i]))
                 {
-                    // return whatever data we may have left in the packet as client data
+                    // data left in the packet seems to do more harm than good, so we are tossing it, but adding to debug log 
                     var remaining = bytesReceived - i - 1;
-                    return (clientData.TakeLast(remaining).ToArray(), remaining);
+                    _logger.Debug($"Tossing extra data \"{System.Text.Encoding.ASCII.GetString(clientData.TakeLast(remaining).ToArray())}\"");
```

Comment:
Would just change this debug to `Ignoring extra rlogin data:`

Reply:
Done